### PR TITLE
Adding data layer and renaming to Primaza

### PR DESCRIPTION
**Pull request by aureamunoz**
Add data layer and rest endpoints to access

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,6 +59,23 @@
...
     </dependency>
+
+    <!-- Data -->
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-orm-rest-data-panache</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
```
Comment (cmoulliard) [inline pom.xml:70]: Can we use h2 instead of postgresql to avoid to launch locally docker desktop ?

File: servicebox-app/pom.xml [hunk 2]
```diff
@@ -36,6 +25,19 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-arc</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-hibernate-orm-rest-data-panache</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-jdbc-postgresql</artifactId>
```
Comment (cmoulliard) [inline servicebox-app/pom.xml:34]: See my previous comment about h2 vs postgresql

**Review by cmoulliard — COMMENTED** [review]
LGTM. See my 2 comments

File: pom.xml (hunk 1 above)
Comment (aureamunoz) [reply pom.xml:70]: yes, let me do it

**Comment by aureamunoz** [comment]
> LGTM. See my 2 comments

done

**Review by cmoulliard — APPROVED** [review]

**Pull request merged**